app/src/androidTest/java/ch/sdp/vibester/activity/profile/MyProfileFragmentTest.kt

```diff
@@ -104,6 +104,26 @@ class MyProfileFragmentTest {
     }
 
 
+    @Test
+    fun checkChangePhotoCancel() {
+        val inputProfile = User( "Lalisa Bon","bit.ly/3IUnyAF", "lisa@test.com",  12, 8, "VvPB47tQCLdjz3YebilS6h5EXdJ3")
```

Comment:
can use R.string.test_profile_image

Reply:
I mean that field is useless now, still here for compatibility reasons. I don't think it is worth it to use context for that old field to retrieve a not used value 😅.